src/main/java/biz/paluch/logging/gelf/log4j/GelfLayout.java

```diff
@@ -0,0 +1,70 @@
+package biz.paluch.logging.gelf.log4j;
+
+import static biz.paluch.logging.gelf.LogMessageField.NamedLogField.*;
+
+import org.apache.log4j.Layout;
+import org.apache.log4j.spi.LoggingEvent;
+
+import biz.paluch.logging.gelf.LogMessageField;
+import biz.paluch.logging.gelf.MdcGelfMessageAssembler;
+import biz.paluch.logging.gelf.intern.GelfMessage;
+
+public class GelfLayout extends Layout {
+
+	private final MdcGelfMessageAssembler gelfMessageAssembler;
+
+	public GelfLayout() {
+		super();
+		gelfMessageAssembler = new MdcGelfMessageAssembler();
+		gelfMessageAssembler.addFields(LogMessageField.getDefaultMapping(Time, Severity, ThreadName, SourceClassName, SourceMethodName,
+				SourceLineNumber, SourceSimpleClassName, LoggerName, NDC));
+	}
+
+	@Override
+	public String format(LoggingEvent loggingEvent) {
+		GelfMessage gelfMessage = gelfMessageAssembler.createGelfMessage(new Log4jLogEvent(loggingEvent));
+		return gelfMessage.toJson("") + System.lineSeparator();
```

Comment:
Have you considered using `Layout#LINE_SEP`? `System.lineSeparator()` is available since Java 1.7.